.github/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@
 - When developing a new feature, include at least one test when applicable.
 - When submitting a PR, please follow [this template](PULL_REQUEST_TEMPLATE.md) (which will probably be already filled up once you create the PR).
 - When submitting a PR with changes to user interface (e.g.: new screen, ...), please add screenshots to the PR description.
-- When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link and place it in your PR.
+- When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/docs/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link and place it in your PR.
```

Comment:
```suggestion
- When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link and place it in your PR.
```

No need to add "docs" to the path, as this is an actual Wiki page. So it does not refer to the wiki file on the repository :)